### ISSUE TITLE
Create visual disabled state on entry list

### ISSUE THREAD

**Issue by devinreams** [mozilla-lockwise/lockwise-ios#364]
For when no content in the list view (or syncing, which is also "no content"):

- [x] sort button should look disabled/inactive (40% opacity) 
- [ ] search/filter should look disabled/inactive (30% opacity, search icon & text placeholder)

## Visual Design
**InVision**
During Syncing: https://mozilla.invisionapp.com/share/NHIMP2NYXT6
No Entries Found: https://mozilla.invisionapp.com/share/8QIMP4O2VAH 

**Zeplin**
During Syncing: https://zpl.io/2Ejdl78
No Entries Found: https://zpl.io/VxKMpdg

**Comment by joeyg** [comment]
There were a couple PRs referencing this issue. Can it be closed?

**Comment by devinreams** [comment]
>There were a couple PRs referencing this issue. Can it be closed?

Yes there were but there is still no "disabled" state for search entries since @sashei deemed some changes to the component were needed. ✋ 

Sort button and settings are/currently have disabled states. ✅

**Comment by changecourse** [comment]
If I remember correctly, @sashei wanted to create a new component that represented the disabled search bar, correct, then just swap between them as needed? 

Just adding my naive context ;)

**Comment by sashei** [comment]
I don't remember exactly what I was thinking here, but what @changecourse says makes sense looking at this issue again... there's not a nice way to just set `disabled` on a `TableViewCell`, we can make it un-tappable but the visual grey-ing out is a little more involved.

**Comment by sandysage** [comment]
@changecourse what should the visual be for the disabled state here now that we've changed the search feature?

Not a priority. Just noting we can't pick this up without a new design here.
